### ISSUE TITLE
Service workers not enabled/supported in current app

### ISSUE THREAD

**Issue by ikreymer** [kiwix/apple#290]
Similar to kiwix/kiwix-desktop#487, the embedded WKWebView does not appear to support service workers.
They would be needed for the new replay system.

It's not entirely clear if WKWebView supports them fully, see:
https://developer.apple.com/forums/thread/126923 it may require a different approach or using Safari directly,
or maybe this has changed.

(I tried the existing app on MacOS, ran into issues when trying to build locally per instructions. Can open separate issue for that)

**Comment by kelson42** [comment]
@automactic This is a critical problem as we plan to put in production ZIM files using service workers in Q4 of this year. Would you be able to give us quickly a feedback on this?

**Comment by kelson42** [comment]
@automactic @ikreymer Should we consider this is the same bug/ticket for iOS and macOS or should we open a separate ticket?

**Comment by rgaudin** [comment]
@kelson42 we can keep a single ticket until this gets implemented as both iOS and macOS uses `WKWebView` (although implemented at different locations).

Service workers (whatever the scheme – both app uses `kiwix://`) won't work on current iOS 13 nor Autumn expected iOS14 as per [this SO answer](https://stackoverflow.com/questions/49673399/service-workers-unavailable-in-wkwebview-in-ios-11-3)

tldr: the feature has been implemented but it requires a *permission* (entitlement) which can not be requested on the store backend.

**Comment by kelson42** [comment]
Basically a duplicate of #341
